src/glass/src/app/shared/services/api/status.service.ts

```diff
@@ -22,11 +22,19 @@ export interface ClusterStatus {
   health: HealthStatus;
 }
 
+export enum StageEnum {
```

Comment:
I would suggest to use `StatusStageEnum` because we also have a `StageEnum` in the `/api/bootstrapt/status` response. Normally it is no problem having to types with the same name in TS because you can rename them on import, but IMO it is better to have unique names if possible.

```suggestion
export enum StatusStageEnum {
```